pkg/config/node.go

```diff
@@ -643,6 +643,22 @@ func getSortedBackends(kubeconfigPath string, readFromLocalAPI bool, vips []net.
 		}).Info("Failed to get master Nodes list")
 		return []Backend{}, err
 	}
+	// When installing TNA clusters using assisted service, one of the master nodes acts as the bootstrap.
+	// So during the installation there will only be one master node, but we need two in order to configure keepalived.
+	// We cannot wait until the bootstrap finishes and becomes a master, because then no node will have the API vip.
+	// To circumvent that we will add the arbiter node to the list of nodes.
+	if len(nodes.Items) == 1 {
```

Comment:
Is keepalived always running on the arbiter? If so, it should always be included in the peer list. Otherwise you run the risk of split brain where the arbiter has keepalived sending announcement packets but the other nodes don't know about it and can't coordinate properly.